compiler/src/iree/compiler/Codegen/Common/LinkTuningSpecsPass.cpp

```diff
@@ -154,6 +398,27 @@ struct LinkTuningSpecsPass final
 FailureOr<NamedSequenceOp> linkTuningSpecs(ModuleOp module) {
   SmallVector<NamedSequenceOp> tuningSpecs;
 
+  int matchingModules = 0;
+  int totalModules = 0;
```

Comment:
I'd put this variable first since you update it unconditionally in the loop below.